tabs/transponder.js

```diff
@@ -1,104 +1,211 @@
 'use strict';
 
-TABS.transponder = {
-    available: false
+var transponderType = {
```

Comment:
remove this, see TRANSPONDER.providers = you can find an i18n message for each provider using the id in the i18n message key.  e.g. 'transponderProvider0', 'transponderProvider1', etc.  There should be NO ilap or arcitimer specific code required when the new MSP_TRANSPONDER_CONFIG is used.